azurerm/resource_arm_subnet.go

```diff
@@ -125,6 +125,11 @@ func resourceArmSubnet() *schema.Resource {
 					},
 				},
 			},
+
+			"private_link_service_network_policies": {
+				Type:     schema.TypeString,
```

Comment:
Should this be a bool?

Reply:
Yes, I changed it... it is actually a string `Enabled` or `Disabled`, however I have exposed this as a `bool` now...